deps/rabbit/src/rabbit_channel_tracking.erl

```diff
@@ -149,27 +130,9 @@ register_tracked_ets(TrackedCh = #tracked_channel{pid = ChPid, username = Userna
     end,
     ok.
 
-register_tracked_mnesia(TrackedCh =
-                            #tracked_channel{node = Node, name = Name, username = Username}) ->
-    ChId = rabbit_tracking:id(Node, Name),
-    TableName = tracked_channel_table_name_for(Node),
-    PerUserChTableName = tracked_channel_per_user_table_name_for(Node),
-    case mnesia:dirty_read(TableName, ChId) of
-      []    ->
-          mnesia:dirty_write(TableName, TrackedCh),
-          mnesia:dirty_update_counter(PerUserChTableName, Username, 1),
-            ok;
-      [#tracked_channel{}] ->
-          ok
-    end,
-    ok.
-
 -spec unregister_tracked_by_pid(pid()) -> any().
 unregister_tracked_by_pid(ChPid) when node(ChPid) == node() ->
-    case rabbit_feature_flags:is_enabled(tracking_records_in_ets) of
-        true -> unregister_tracked_by_pid_ets(ChPid);
-        false -> unregister_tracked_by_pid_mnesia(ChPid)
-    end.
+    unregister_tracked_by_pid_ets(ChPid).
```

Comment:
Let's delete `unregister_tracked_by_pid_ets/1` and inline its function body directly into `unregister_tracked_by_pid/1`